src/schedule/ll_schedule.c

```diff
@@ -331,7 +331,7 @@ static void schedule_ll_domain_clear(struct ll_schedule_data *sch,
 	/* unregister the task */
 	domain_unregister(domain, task, atomic_read(&sch->num_tasks));
 
-	tr_info(&ll_tr, "num_tasks %d total_num_tasks %d",
+	tr_info(&ll_tr, "num_tasks %ld total_num_tasks %ld",
 		atomic_read(&sch->num_tasks),
 		atomic_read(&domain->total_num_tasks));
```

Comment:
strictly speaking, this file isn't used with Zephyr so these changes aren't fixing Zephyr builds directly, but it's true, that we need consistency here. OTOH using `%ld` with 32-bit **should** actually generate warnings like
```
/tmp/p.c: In function ‘main’:
/tmp/p.c:8:19: warning: format ‘%ld’ expects argument of type ‘long int’, but argument 2 has type ‘int32_t’ {aka ‘int’} [-Wformat=]
    8 |  return printf("%ld\n", x);
      |                 ~~^     ~
      |                   |     |
      |                   |     int32_t {aka int}
      |                   long int
      |                 %d
```
in XTOS builds but since we don't use `__attribute__((__format__(printf,... ,...)))` with XTOS builds (so far) we shouldn't be getting these. So, so far this should be safe...